composer/init.sls

```diff
@@ -0,0 +1,23 @@
+include:
+  - jcu.php
+  - jcu.php.json
+  - jcu.php.intl
+  - jcu.php.xml
+
+get-composer:
+  cmd.run:
+    - name: 'CURL=`which curl`; $CURL -sS https://getcomposer.org/installer | php'
+    - unless: test -f /usr/local/bin/composer
+    - cwd: /root/
+    - require:
+      - pkg: php
+      - pkg: php-json
+      - pkg: php-intl
+      - pkg: php-xml
```

Comment:
See if these other pkgs are required; my original setup just needed `php` but EL8 might be different.

Reply:
If they are not here then the composer build step doesn't work.

Reply:
Ah, I meant the `php-json`, `php-intl` and `php-xml` specifically.  I just checked quickly and `php-json` is definitely needed to install but composer seems to be okay without `php-intl` and `php-xml`.

Reply:
I'll have another look into this and see why I have added them and write back.

Reply:
Had a look again, those two are required for the libraries I'm installing with `composer`, and not necessary for `composer` itself. So I've removed them from the shared salt states and I'll add them with an `extend` on the `journals` project.